### PR TITLE
OWC/2016: correct match links

### PR DESCRIPTION
**Pull request by MegaApplePi**
as reported in <https://osu.ppy.sh/community/forums/topics/630416>

### CODE REVIEW

File: wiki/Tournaments/OWC/2016/en.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ Sunday, 13. November 2016
 1. The osu! World Cup is a country-based team tournament, played on the osu! game mode.
    * This competition is planned as a 4 versus 4 setup.
 2. Map scoring is based on Score V2.
-3. The maps for each round will be announced by the tournament management in advance on the weekend before the actual matches take place. Only these will be used during the respective matches.
+3. The maps for each round will be announced by the tournament management in advance on the weekend before the actual community/matches take place. Only these will be used during the respective community/matches.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/OWC/2016/en.md]: Hi. CTRL-R;Replace All problem here.

File: wiki/Tournaments/OWC/2016/en.md [hunk 2]
```diff
@@ -477,7 +477,7 @@ Sunday, 13. November 2016
 16. All players and referees must to be treated with respect. Instructions of the referees and tournament management is to be followed. Decisions labeled as final are not to be objected.
 17. Disrupting the match by foul play, picking inappropriate warmup maps (see below), insulting and provoking other players or referees, delaying the match or other deliberate inappropriate misbehavior is strictly prohibited.
 18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules "Rules"). All chat rules apply to the multiplayer chatrooms, too.
-    1. Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.
+    1. Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer community/matches and must be exchanged for the time being.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/OWC/2016/en.md]: ditto

File: wiki/Tournaments/OWC/2016/en.md [hunk 3]
```diff
@@ -509,7 +509,7 @@ Sunday, 13. November 2016
    1. This may change according to the final amount of formed teams.
 2. All the teams from each group will face each other.
 3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
-   1. Most matches won.
+   1. Most community/matches won.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/OWC/2016/en.md]: ditto

File: wiki/Tournaments/OWC/2016/en.md [hunk 4]
```diff
@@ -570,9 +570,9 @@ Sunday, 13. November 2016
 ### Scheduling Instructions
 
 1. Each stage will be held on a single weekend.
-2. Matches in Group Stage may overlap.
-3. All matches will be held on either Saturday or Sunday.
-4. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first matches of the actual stage. Tournament Management will try to create the schedule to respect the participant's time zone.
+2. community/matches in Group Stage may overlap.
+3. All community/matches will be held on either Saturday or Sunday.
+4. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first community/matches of the actual stage. Tournament Management will try to create the schedule to respect the participant's time zone.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/OWC/2016/en.md]: ditto

File: wiki/Tournaments/OWC/2016/en.md [hunk 5]
```diff
@@ -570,7 +570,7 @@ Sunday, 13. November 2016
 ### Scheduling Instructions
 
 1. Each stage will be held on a single weekend.
-2. Matches in Group Stage may overlap.
+2. matches in Group Stage may overlap.
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/2016/en.md]: ?

**Pull request merged**